config/locales/devise.en.yml

```diff
@@ -60,14 +67,3 @@ en:
       not_saved:
         one: "1 error prohibited this %{resource} from being saved:"
         other: "%{count} errors prohibited this %{resource} from being saved:"
-en:
```

Comment:
💁 This was redefining all the devise messages, so I moved them to the appropriate places in the configuration.